sql/core/src/test/scala/org/apache/spark/sql/UserDefinedTypeSuite.scala

```diff
@@ -277,4 +279,12 @@ class UserDefinedTypeSuite extends QueryTest with SharedSparkSession with Parque
     val udt = new TestUDT.MyDenseVectorUDT()
     assert(!Cast.canUpCast(udt, StringType))
   }
+
+  test("SPARK-29961 typeof user defined type") {
```

Comment:
You don't need `SPARK-29961` in the prefix: https://github.com/apache/spark-website/commit/d7aa99e926e3748928e2ddc244369518dad3216b